lib/transformations/defineTest.js

```diff
@@ -13,10 +13,6 @@
 
 const fs = require('fs');
 const path = require('path');
-const cli_engine = require('eslint').CLIEngine;
-const eslintrules = require(process.cwd() + '/.eslintrc.json');
-eslintrules.fix = true;
-const cli = new cli_engine(eslintrules);
```

Comment:
I did this to counteract [this issue](https://github.com/benjamn/recast/issues/242) with recast. But we are using a branch of recast right now, until this issue is fixed. So thanks for cleaning this up.